static/public/index.html

```diff
@@ -39,9 +71,24 @@
             <!-- Empty nav for now -->
         </nav>
         
-        <div class="flex items-stretch w-48">
+        <!-- Desktop GitHub button -->
+        <div class="desktop-nav flex items-stretch">
```

Comment:
i think this would always show the github button, even if we show the nav. honestly for this one we prolly dont need the mobile nav so much as just descreasing the button margin for smaller screens